tests/integration-test/src/tests/utils.rs

```diff
@@ -251,7 +256,12 @@ pub fn add_tracepoint(globals: Option<Vec<&str>>, load_type: &LoadType) -> Resul
 }
 
 /// Attach a uprobe program to bpfctl with bpfctl
-pub fn add_uprobe(globals: Option<Vec<&str>>, load_type: &LoadType) -> Result<String> {
+pub fn add_uprobe(
```

Comment:
Just thinking.  Now that you've parameterized the image/file stuff, we could combine the add_*probe and add_*retprobe functions and have a boolean retprobe parameter.  That can be another pr, though.

Reply:
++